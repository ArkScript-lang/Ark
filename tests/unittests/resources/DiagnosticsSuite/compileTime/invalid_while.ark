(while ($ a 1) {
    (mut acc (+ acc (@ [] a)))})
