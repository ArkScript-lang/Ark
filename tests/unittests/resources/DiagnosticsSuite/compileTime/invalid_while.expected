At (while (begin (mut acc (+ acc (@ (list) 1))))) @ 1:1
    1 | (while ($ a 1) {
      | ^~~~~~~~~~~~~~~~
    2 |     (mut acc (+ acc (@ [] a)))})
      | ^~~~~~~~~~~~~~~
    3 |
        Invalid node ; if it was computed by a macro, check that a node is returned
